.github/workflows/unit-tests.yml

```
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9]
```

```suggestion
        python-version: [3.7, 3.8, 3.9, 3.10]
```

I think we thought we would replace 3.6 with 3.10 instead?